code/modules/clothing/head/hardhat.dm

```diff
@@ -1,15 +1,29 @@
+/*
+
+	Contents: 
+
+		Hard hat
+		Orange hard hat
+		Firefighter helmet
+		White hard hat (CE's hard hat)
+		Blue hard hat
+		Atmos tech's firefighter helmet
+		Welding hard hat
+		Welding organe hard hat
+		Welding white hard hat
+
+*/
+
 /obj/item/clothing/head/hardhat
 	name = "hard hat"
 	desc = "A piece of headgear used in dangerous working conditions to protect the head. Comes with a built-in flashlight."
 	icon_state = "hardhat0_yellow"
 	item_state = "hardhat0_yellow"
-	var/hat_type = "yellow" //Determines used sprites: hardhat[on]_[hat_type] and hardhat[on]_[hat_type]2 (lying down sprite)
 	armor = list("melee" = 15, "bullet" = 5, "laser" = 20, "energy" = 10, "bomb" = 20, "bio" = 10, "rad" = 20, "fire" = 100, "acid" = 50, "stamina" = 20)
-	flags_inv = 0
+	flags_inv = NONE
 	actions_types = list(/datum/action/item_action/toggle_helmet_light)
 	resistance_flags = FIRE_PROOF
 	clothing_flags = SNUG_FIT
-	dynamic_hair_suffix = "+generic"
```

Comment:
Why's this being removed?

Reply:
Because its already set to `+generic` on the parent